lua/harpoon/buffer.lua

```diff
@@ -73,6 +73,20 @@ function M.setup_autocmds_and_keymaps(bufnr)
         "<Cmd>lua require('harpoon.buffer').run_select_command()<CR>",
         {}
     )
+    vim.api.nvim_buf_set_keymap(
+        bufnr,
+        "n",
+        "<C-x>",
+        "<Cmd>lua require('harpoon.buffer').run_select_command({ split = true })<CR>",
+        {}
+    )
+    vim.api.nvim_buf_set_keymap(
+        bufnr,
+        "n",
+        "<C-v>",
+        "<Cmd>lua require('harpoon.buffer').run_select_command({ vsplit = true })<CR>",
+        {}
+    )
```

Comment:
You could use the `vim.keymap.set` that accepts lua functions too
```suggestion
    vim.keymap.set(
        "n",
        "<C-v>",
        function() require("harpoon.buffer").run_select_command({ vsplit = true }) end,
        { buffer = bufnr }
    )
```